processes.md

```diff
@@ -0,0 +1,201 @@
+
+To facilitate productivity, small repositories, repositories that do not contain production code or that have a low traffic of PRs may decide to use simpler requirements. Each of these projects may define their own requirements, in a local `processes.md` file.
+
+Process of becoming an Maintainer:
+
+1. Any current Maintainer may nominate a current [Reviewer](#Reviewer) to become a new Maintainer, by opening a PR against the appropriate project, adding the contributor to `MAINTAINERS.md`
+2. The nominee will add a comment to the PR testifying that they agree to all requirements of becoming a Maintainer.
+3. A majority of the current Maintainers must then approve.
+4. Add the new maintainer to the corresponding GitHub team
+
+- Each project has a `<repo>-maintainers` team, where `<repo>` is the name of the GitHub repository.
+
+## Governance Committee Member
+
+Description: The CDEvents Governance committee is the governing body of the CDEvents open source project. It's an elected group that represents the contributors to the project, and has an oversight on governance and technical matters.
+
+See [governance.md](/governance.md) for requirements, responsibilities, and election process.
+
+- Additional privileges:
+  - Maintainer privileges on all CDEvents projects
+  - Organization admin access.
+
+Members of the governance committee belong to the the `governance` GitHub team.
+
+## Inactivity
+
+It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+- Inactivity is measured by:
+  - Failing to meet role requirements.
+  - Periods of no [contributions](./README.md#contributions) for longer than 4 months
+  - Periods of no communication for longer than 2 months
+- Consequences of being inactive include:
+  - Involuntary removal or demotion
+  - Being asked to move to Emeritus status
+
+### Involuntary Removal or Demotion
+
+Involuntary removal/demotion of a contributor happens when responsibilities and requirements aren't being met. This may include repeated patterns of inactivity, extended period of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
+
+Involuntary removal or demotion is handled through a vote by a majority of the [CDEvents Governing Board](/governance.md).
+
+### Stepping Down/Emeritus Process
+
+If and when contributors' commitment levels change, contributors can consider stepping down (moving down the contributor ladder) vs moving to emeritus status (completely stepping away from the project).
+
+Contact the Maintainers about changing to Emeritus status, or reducing your contributor level.
```

Comment:
Can one become an Emeritus and then come back? If so do we need to write something about it?

Reply:
But I was referring to the contribution ladder not the governing board.